geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/StatParser.scala

```diff
@@ -66,8 +67,24 @@ private class StatParser extends BasicParser {
     oneOrMore(singleStat, ";") ~~> { s => if (s.length == 1) s.head else new SeqStat(s) } ~ EOI
   }
 
+  private def groupBy: Rule1[Stat] = rule {
+    "GroupBy(" ~ string ~ "," ~ oneOrMore(statString, ";") ~ ")" ~~> { (attribute, groupedStatsList) =>
```

Comment:
I'd like to take a crack at using the `~~~?` operator here, but for now this should be ok